app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
![image](https://user-images.githubusercontent.com/105493988/218831513-ef479c95-663e-47b2-a82e-c6229cbda4e8.png)
Fix everywhere, according to the task

Reply:
done